tests/push.bats

```diff
@@ -45,7 +45,7 @@ load helpers
   _prefetch alpine
   run_buildah from --quiet --pull=false $WITH_POLICY_JSON alpine
   cid=$output
-  run_buildah push $WITH_POLICY_JSON --format oci alpine dir:$mytmpdir
+  run_buildah push --retry 4 --retry-delay 4s $WITH_POLICY_JSON --format oci alpine dir:$mytmpdir
```

Comment:
I don't think any of these tests that you have modified will normally retry 4 times.  I think they'll each succeed on the first pull. It would be nice to have at least one test somewhere that does retry 4 times every time and that the time to do so is at least 16 seconds.

Reply:
This is only to verify if flags are treated as valid flags and buildah does not end up not parsing these flags, testing retry in CI is really hard since its hard to simulate registry failure. So these are just added for sanity and another detailed parsing test in `tests/bud.bats`.